package.json

```diff
@@ -50,17 +50,19 @@
     ]
   },
   "peerDependencies": {
-    "@fortawesome/fontawesome-svg-core": "~1 || ~6",
-    "react": ">=16.3"
+    "@fortawesome/fontawesome-svg-core": "~1 || ~6 || ~7 || ^7.0.0-alpha8",
```

Comment:
@robmadole ---
For the svg core... I updated to support 7 and the alpha.

But should we be allowing minor patch updates... from my understanding the `~` will not allow minor updates.
So my thinking is that we should update this line to:
```
"@fortawesome/fontawesome-svg-core": "^1 || ^6 || ^7 || ^7.0.0-alpha8",
```
... to allow minor patches as well.

What are your thoughts?